Header header
float64[6] relative_movement
bool grasp